CHANGES.txt

```diff
@@ -1,3 +1,8 @@
+2.0.0 - 2016-01-16
+==================
+
+ * Py27, Py35 compliant
```

Comment:
spelling nit: "compliant"

Reply:
Hi @rfk , sorry for my delay .. so much work to do :-0

Thanks for the hint, I will fix it.

Reply:
> sorry for my delay .. so much work to do

No worries at all!  Thanks for helping to push this through.